@ankitmathur-db one  MLflow tests fails. Looks like an RST error. 

```
make: *** [rsthtml] Error 1
Makefile:70: recipe for target 'rsthtml' failed
##[error]Process completed with exit code 2.
```

--- a/docs/source/model-registry.rst
+++ b/docs/source/model-registry.rst
@@ -46,7 +46,7 @@ UI Workflow
 -----------
 
 Registering a Model
-^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^
 
 #. From the MLflow Runs detail page, select a logged MLflow Model in the **Artifacts** section.
 

